src/pages/index.js

```diff
@@ -140,7 +140,7 @@ export default function Home() {
                 </p>
                 <p>
                   Cadence includes several language features that prevent entire classes of bugs via a strong static type system, <a href="https://developers.flow.com/cadence/language/functions#function-preconditions-and-postconditions">design by contract</a>,
-                  and <a href="https://developers.flow.com/cadence/language/capability-based-access-control">capability-based access control</a>.
+                  and <a href="https://developers.flow.com/cadence/language/access-control">capability-based access control</a>.
```

Comment:
The "Access Control" page describes access control functionality of code, e.g. the `access` keyword.

This should link to the capabilities page:

```suggestion
                  and <a href="https://cadence-lang.org/docs/language/capabilities">capability-based access control</a>.
```